app/settings/globals.py

```diff
@@ -39,9 +39,12 @@
 ENV = config("ENV", cast=str, default="dev")
 
 DATA_LAKE_BUCKET = config("DATA_LAKE_BUCKET", cast=str, default=None)
+
 TILE_CACHE_BUCKET = config("TILE_CACHE_BUCKET", cast=str, default=None)
 TILE_CACHE_CLOUDFRONT_ID = config("TILE_CACHE_CLOUDFRONT_ID", cast=str, default=None)
 TILE_CACHE_URL = config("TILE_CACHE_URL", cast=str, default=None)
+TILE_CACHE_CLUSTER = config("TILE_CACHE_CLUSTER", cast=str, default=None)
+TILE_CACHE_SERVICE = config("TILE_CACHE_CLUSTER", cast=str, default=None)
```

Comment:
Just checking, these are meant to be the same, right?

Reply:
no, this should be different, thanks for catching